docker/lite/Dockerfile

```diff
@@ -45,7 +42,7 @@ RUN groupadd -r vitess && useradd -r -g vitess vitess
 RUN mkdir -p /vt/vtdataroot && chown -R vitess:vitess /vt
 
 # Set up Vitess environment (just enough to run pre-built Go binaries)
-ENV VTROOT /vt/src/vitess.io/vitess
+ENV VTROOT /vt
```

Comment:
I believe this was wrong to begin with, leading to the error: `bash: vtgate: command not found`:
```
$ docker run -it --user=vitess vitess/lite:v20.0.0 bash
vitess@0d7c6ece9106:/$ vtgate
bash: vtgate: command not found
vitess@0d7c6ece9106:/$
```

With this fix, the vitess binaries are directly in the `PATH` and people can execute them without specifying the full path to the binary.

Reply:
With the fix:

```
$ docker run -it docker.io/vitess/lite-test:latest bash     
vitess@6a3782014166:/$ vtgate --version 
vtgate version Version: 21.0.0-SNAPSHOT (Git revision 694a0cf8d9e51aa4b130ad1656c4009427eb0e3d branch 'remove-bootstrap-from-lite') built on Thu Jul  4 15:55:37 UTC 2024 by vitess@buildkitsandbox using go1.22.5 linux/amd64
vitess@6a3782014166:/$
```